bosh-shell/startup.sh

```diff
@@ -2,25 +2,26 @@
 
 set -eu
 
-if [ -z "${BOSH_ID_RSA:-}" ] || [ -z "${BOSH_IP:-}" ] || [ -z "${BOSH_ENVIRONMENT:-}" ] || [ -z "${BOSH_CA_CERT:-}" ]; then
-  echo "WARNING: You need to set BOSH_ID_RSA, BOSH_IP, BOSH_ENVIRONMENT and BOSH_CA_CERT."
+if [ -z "${USER_ID_RSA:-}" ] || [ -z "${BOSH_ID_RSA:-}" ] || [ -z "${BOSH_IP:-}" ] || [ -z "${BOSH_ENVIRONMENT:-}" ] || [ -z "${BOSH_CA_CERT:-}" ]; then
```

Comment:
Do we still need BOSH_ID_RSA as well here? Looks like it's only going to use the USER_ID_RSA one.

Reply:
Was planning on removing it in a follow up PR after a paas-cf deployment

Reply:
We reference the hash explicitly, so breaking changes are fine :)